### PR TITLE
Added Add remove todo item support

### PR DESCRIPTION
**Pull request by rebel216**
Added add and remove functionality.

### CODE REVIEW

File: src/Remove.js [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+        checkBtn[i].setAttribute('data-id', i);
+        checkMark[i].setAttribute('data-id', i);
+        dotsIcon[i].setAttribute('data-id', i);
+        trashIcon[i].setAttribute('data-id', i);
+        label[i].setAttribute('for', i);
+        textFields[i].setAttribute('id', i);
+    }
+    localStorage.setItem('toDoList', JSON.stringify(unsortedItems));
+    getStatus();
+}
```
Comment (EricMbouwe) [inline src/Remove.js:37]: - [ ] By default the property index should be set to the value of the new array length (i.e. if a 5th task is added to the list, the index of that task should equal 5). Currently, your base index is `0`. Let's fix that

File: src/Remove.js [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+            listItem.style.backgroundColor = 'lightyellow';
+            field.style.backgroundColor = 'lightyellow';
+            dotsIcon.classList.add('hide');
+            trashIcon.classList.remove('hide');
+            // Delete function
+            trashIcon.addEventListener('mousedown', () => {
+                const storedItems = getStatus();
+                listItem.remove();
+                storedItems.splice(fieldId, 1);
+                localStorage.setItem('toDoList', JSON.stringify(storedItems));
```
Comment (EricMbouwe) [inline src/Remove.js:60]: - [ ] This action removes more than it should, and your local storage is not well updated and your list view. I would advise using a `filter` function here and the data-id of the target tod to remove it.

**Review by EricMbouwe — CHANGES_REQUESTED** [review]
Hi @rebel216, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- Good Styling
- Linters Ok

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- x
- y
- z

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by AshakaE — APPROVED** [review]
Hi @rebel216 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**